### DISCUSSION TITLE
About Gas

### DISCUSSION THREAD

**Discussion by MartinsAloysius** [Cyfrin/foundry-full-course-cu#2691] [Q&A]
Started the class on gas, Mr. Patrick said to find out the amount of txn fee, we multiply the amount of gas by the gas price. How do we know these values when we haven't done the txn.

**Comment by EngrPips** [comment]
You can simulate your transaction on a local chain such as `Anvil` and get the amount of gas the transaction would cost. Then you can look up the website such as https://milkroad.com/ethereum/gas/ to know what is the current gas price on the Ethereum blockchain and like that, you can have some idea of how much your transaction might cost.
